packages/liveblocks-react-ui/src/primitives/ErrorBoundary.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ComponentType, ErrorInfo, ReactNode } from "react";
```

Comment:
Do you think we'll export this publicly? If so, I could see it in `@liveblocks/react` like `ClientSideSuspense`, otherwise maybe we could move it in `/src/utils` here rather than `/src/primitives` like other private utilitarian components, https://github.com/liveblocks/liveblocks/blob/main/packages/liveblocks-react-ui/src/utils/Portal.tsx for example.

Reply:
Maybe! But yeah, I didn't know where to best put it. I'll move it to utils for now 👍